### PR TITLE
Git-Bash_Anupriya 

### PR DESCRIPTION
**Pull request by anupriyakkumari**
Problem 4 pending. Rest are done. All are in README.

### CODE REVIEW

**Comment by guptaaryan16** [comment]
@anupriyakkumari nicely done !!! Liked your api for Kaggle but it would be better if you provided some default command to be entered
@divo12 request to merge

File: Anupriya/Git-Bash_Anupriya/README.md [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+```
+
+```
+git cat-file -p 3fa0d4b98289a95a7c
+o/p :
+Hello, World
+```
+
+## Problem 4
+
```
Comment (guptaaryan16) [inline Anupriya/Git-Bash_Anupriya/README.md:475]: @anupriyakkumari can you add the Kaggle API commands here and provide some sample usage of the same

**Comment by guptaaryan16** [comment]
Hey @anupriyakkumari you should create a new branch for the new lecture and then make a PR for the same. SO please remove these changes from this current PR. You can use `git revert` for the same and add the new changes to a separate branch on GitHub

File: Anupriya/Git-Bash_Anupriya/README.md (hunk 1 above)
Comment (anupriyakkumari) [reply Anupriya/Git-Bash_Anupriya/README.md:475]: Done

**Comment by guptaaryan16** [comment]
can you remove pre lecture resources 1 for now and then add them later

**Comment by divo12** [comment]
@anupriyakkumari I am merging your PR for now since it doesn't cause any conflicts but from next time make a PR from another branch if you have to submit another assignment before the former assignment is reviewed.

**Comment by anupriyakkumari** [comment]
Okay. Got it!

**Pull request merged**